### DISCUSSION TITLE
Coordinates with multiaction

### DISCUSSION THREAD

**Discussion by mekhaelt** [elgatosf/elgato#52] [Help] [answered]
I currently have a custom plugin that changes the display of all buttons using the custom action based upon preexisting data that changes everyday. It works fine, however I want to modify my streamdeck to add the plugin in a multi-action, however when I do that the coordinates are undefined. Does anyone know what is causing this/a way around this.

**Comment by ChekTek** [comment, marked as answer]
Hi @mekhaelt,

Apologies for the delayed response. At this point in time, coordinates are not provided in event while in a multi-action. We are actively looking at adding this information to multi-action events.
